### PR TITLE
fix(ui): TE-2284 section headers for template properties

### PR DESCRIPTION
**Pull request by harshilvelotio**
#### Issue(s)

https://startree.atlassian.net/browse/TE-2284

#### Description

- Added section headers for each template properties

#### Screenshots
image
image

#### How to test
- Go to "Create Alert"
- Switch to Advanced view

### CODE REVIEW

**Review by cyrilou242 — CHANGES_REQUESTED** [review]
added notes in jira: 
https://startree.atlassian.net/browse/TE-2284?focusedCommentId=30205

File: thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template-properties-builder/alert-template-properties-builder.component.tsx [hunk 1]
```diff
@@ -32,14 +32,26 @@ import React, {
...
 import { useAlertWizardV2Styles } from "../../alert-wizard-v2.styles";
-import { AlertTemplateFormField } from "../alert-template-form-field/alert-template-form-field.component";
 import { setUpFieldInputRenderConfig } from "../alert-template.utils";
 import {
     AlertTemplatePropertiesBuilderProps,
     PropertyRenderConfig,
 } from "./alert-template-properties-builder.interfaces";
+import { AlertTemplateSectionedProperties } from "./alert-template-sectioned-properties/alert-template-sectioned-properties.component";
+
+// const PropertySectionHeader = withStyles<
```
Comment (alcatraz627) [inline thirdeye-ui/src/app/components/alert-wizard-v2/alert-template/alert-template-properties-builder/alert-template-properties-builder.component.tsx:45]: Do we want to keep this code or remove it?

**Review by alcatraz627 — APPROVED** [review]
Looks fine overall, added a couple of code cleanup comments

**Review by cyrilou242 — APPROVED** [review]
feature wise LGTM

**Pull request merged**